xbmc/settings/lib/SettingSection.cpp

```diff
@@ -78,6 +81,20 @@ bool CSettingGroup::Deserialize(const TiXmlNode *node, bool update /* = false */
   if (!ISetting::Deserialize(node, update))
     return false;
 
+  const TiXmlElement *element = node->ToElement();
+  if (element == NULL)
+    return false;
+
+  int tmp = -1;
+  if (element->QueryIntAttribute(SETTING_XML_ATTR_LABEL, &tmp) == TIXML_SUCCESS && tmp > 0)
+    m_label = tmp;
+  if (element->QueryIntAttribute(SETTING_XML_ATTR_HELP, &tmp) == TIXML_SUCCESS && tmp > 0)
+    m_help = tmp;
+  if (element->QueryIntAttribute(SETTING_XML_ATTR_BEFORE, &tmp) == TIXML_SUCCESS && tmp > 0)
```

Comment:
Thanks for the feedback. I'm currently doing a rework. Good to have it separated from DSP :-) .
Is <code>#define SETTING_XML_ATTR_HIDE_SEPARATOR "hideseparator"</code> OK for you and not to long.
Sorry for few bad parts on it, have not recorded all the changes.